src/helpers/makeQuery.js

```diff
@@ -37,7 +37,12 @@ const makeQuery = <P: mixed, M: mixed>(
         (node) => isNodeValid(node) && matcherFn(node.props[name], matcher)
       );
     } catch (error) {
-      throw new ErrorWithStack(prepareErrorMessage(error), getBy);
+      throw new ErrorWithStack(
+        `${prepareErrorMessage(error)} with ${name} '${
+          typeof matcher === 'string' ? matcher : JSON.stringify(matcher) || ''
+        }'`,
```

Comment:
What is the reason for using `JSON.stringify` instead of `toString` like in other places?

Reply:
I'm getting this error if I use `toString()`

![image](https://user-images.githubusercontent.com/1479283/94341987-f6195f00-ffdb-11ea-8260-79450c1621f7.png)

```
Cannot call `matcher.toString` because property `toString` is missing in  mixed
```